### PR TITLE
Expose PrefetchSize option which maps to TotalCacheSizeInBytes 

### PR DESCRIPTION
**Pull request by christothes**
This change is strictly for partner team testing and will remain in a feature branch until it is fully vetted.

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs [hunk 1]
```diff
@@ -260,7 +260,8 @@ private static async Task<EventHubScope> BuildScopeFromExistingEventHub()
             using (var client = new EventHubManagementClient(AzureResourceManagerUri, new TokenCredentials(token)) { SubscriptionId = EventHubsTestEnvironment.Instance.SubscriptionId })
             {
                 var eventHub = new Eventhub(partitionCount: partitionCount);
-                eventHub = await ResourceManager.CreateRetryPolicy<Eventhub>().ExecuteAsync(() => client.EventHubs.CreateOrUpdateAsync(resourceGroup, eventHubNamespace, CreateName(), eventHub)).ConfigureAwait(false);
+                //eventHub = await ResourceManager.CreateRetryPolicy<Eventhub>().ExecuteAsync(() => client.EventHubs.CreateOrUpdateAsync(resourceGroup, eventHubNamespace, CreateName(), eventHub)).ConfigureAwait(false);
```
[the remark this answers is not in the document]
Comment (christothes) [reply sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs:263]: no, not intentional.

File: sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/ReadEventOptions.cs [hunk 2]
```diff
@@ -165,6 +168,32 @@ public int PrefetchCount
             }
         }
 
+        /// <summary>
+        ///   The number of events that will be eagerly requested from the Event Hubs service and queued locally without regard to
```
[the remark this answers is not in the document]
Comment (christothes) [reply sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/ReadEventOptions.cs:172]: Agreed, this was basically a placeholder copy from PrefetchCount.

**Comment by jsquire** [comment]
This has been superseded by #15141, closing this out

**Pull request closed without merging**